tools/postinstall-android.js

```diff
@@ -88,7 +91,7 @@ if (data.indexOf("SalesforceHybrid") < 0)
     const oldAndroidDepTree = "android {";
     const newAndroidDepTree = "android {\n\tpackagingOptions {\n\t\texclude 'META-INF/LICENSE'\n\t\texclude 'META-INF/LICENSE.txt'\n\t\texclude 'META-INF/DEPENDENCIES'\n\t\texclude 'META-INF/NOTICE'\n\t}";
     replaceTextInFile(path.join(appProjectRoot, 'app', 'build.gradle'), oldAndroidDepTree, newAndroidDepTree);
-    const newLibDep = "api 'com.salesforce.mobilesdk:SalesforceHybrid'";
+    const newLibDep = "api 'com.salesforce.mobilesdk:SalesforceHybrid:11.0.1'";
```

Comment:
And, here we have a version for the app to use from Maven Central.  The version is ignored when the composite build is present.

Reply:
We should modify https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin/blob/dev/setversion.sh to update that line. We run setversion.sh during releases to update version numbers that are in various files.